libs/scripts/scripts.js

```diff
@@ -124,7 +124,7 @@ const config = {
   fallbackRouting: 'on',
   links: 'on',
   imsClientId: 'milo',
-  imsScope: 'AdobeID,openid,gnav',
+  imsScope: 'AdobeID,openid,gnav,pps.read',
```

Comment:
The IMS scope has been requested and approved for all Milo IMS client IDs. I have checked on both STAGE and PROD and the scope is present.